src/components/Header/Header.js

```diff
@@ -1,12 +1,30 @@
 import React from 'react';
 import HeaderNav from './HeaderNav/HeaderNav';
+import HeaderUserMenu from './HeaderUserMenu/HeaderUserMenu';
+import headerData from '../../headerData.json';
+import { ReactComponent as DopplerLogo } from '../../doppler-logo.svg';
 
-const Header = (props) => {
-  return (
-    <header>
-      <HeaderNav user={props.user} />
-    </header>
-  );
-};
+class Header extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      user: {},
```

Comment:
I think that it should be

```suggestion
      user: null,
```

I mean, when the _user_ is not defined, _null_ is a better representation than an empty object.

Also, empty object will not work with `const isLoggedIn = !!this.state.user;`

Reply:
yes you right